cli/cmd/initnodes.go

```diff
@@ -143,26 +144,16 @@ func InitNodes(ctx context.Context, cfg InitConfig) error {
 		return err
 	}
 
-	if cfg.NodeSnapshot {
-		if err := downloadSnapshot(ctx, cfg.Network, cfg.Home, gethClientName); err != nil {
-			return err
-		}
-
-		if err := downloadSnapshot(ctx, cfg.Network, cfg.Home, haloClientName); err != nil {
-			return err
-		}
+	if err = maybeDownloadSnapshots(ctx, cfg); err != nil {
+		return err
 	}
 
-	if err := maybeDownloadGenesis(ctx, cfg.Network); err != nil {
+	if err = maybeDownloadGenesis(ctx, cfg.Network); err != nil {
```

Comment:
no need to change existing code like this. try to touch the least amount of code in PR. This is in anycase a subjective thing (err var shadowing isn't a problem), no point in changing to your preferred subjective style, because by that logic, other devs will just change it back again.